teslatoken.py

```diff
@@ -39,33 +42,47 @@ def main():
 
     headers = {'client_id': client_data['OWNERAPI_CLIENT_ID']}
 
+    request_url = 'https://owner-api.teslamotors.com/oauth/token';
+    acquire_token = True
     if args.username and args.password:
         headers.update({'grant_type': 'password', 'email': args.username, 'password': args.password})
         headers.update({'client_secret': client_data['OWNERAPI_CLIENT_SECRET']})
+    elif args.username and args.credential_file:
+        with open(args.credential_file) as f:
+            credentials = f.readline().strip()
+        headers.update({'grant_type': 'password', 'email': args.username, 'password': credentials})
+        headers.update({'client_secret': client_data['OWNERAPI_CLIENT_SECRET']})
     elif args.refresh:
         headers.update({'grant_type': 'refresh_token', 'refresh_token': args.refresh})
+    elif args.revoke and args.access_token:
+        request_url = 'https://owner-api.teslamotors.com/oauth/revoke';
+        acquire_token = False
+        headers.update({'token': args.access_token})
     else:
         error('Either username/password or refresh token must be provided.')
```

Comment:
Update this to include something like 'Either username/password or token to refresh/revoke must be provided.'